stable/apm-server/templates/daemonset.yaml

```diff
@@ -52,15 +52,9 @@ spec:
           - name: http
             containerPort: 8200
         livenessProbe:
-          httpGet:
-            path: /
-            port: http
-          initialDelaySeconds: 60
+{{ toYaml .Values.livenessProbe | indent 10 }}
```

Comment:
The use of `nindent` here would make it even more readable:

```
        livenessProbe:
          {{ toYaml .Values.livenessProbe | nindent 10 }}
```

Reply:
Agreed, but then there will be inconsistencies with the other definitions as they are using `indent`. I could change all of them too, but I think that should go in a different PR

Reply:
If you have some time, please just change things you've introduced in this PR to use `nindent` instead.

Changes for other parts should definitely go into a different PR!